packages/package-json-lint/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ## 1.1.0 - (August 31, 2021)
 
+* Added
+  * Added new rule `require-no-hard-coded-dependency-versions`.
+
 * Added
   * Added `projectType` to allow users to flag project as `application`, `module` or `devModule`.
```

Comment:
```suggestion
* Added
  * Added new rule `require-no-hard-coded-dependency-versions`.
  * Added `projectType` to allow users to flag project as `application`, `module` or `devModule`.
```

Reply:
The changelog entry should be above in the unreleased section. Probably just an auto merge error.

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/704/commits/b3446c5a790a1172c0c86da818d7d21076835b53.